authors/apps/authentication/tests/test_registration.py

```diff
@@ -16,13 +20,86 @@ def test_user_register_successful(self):
                                     data=json.dumps(
                                         valid_register_data),
                                     content_type='application/json')
-
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         self.assertEqual(response.data["email"],
                          valid_register_data["user"]["email"])
         self.assertEqual(response.data["username"],
                          valid_register_data["user"]["username"])
 
+    def test_activation_email_is_sent_on_valid_registration(self):
+        """
+        Test activation email is sent to user email upon
+        succesful registration.
+        """
+        self.register_test_user()
+        self.assertEqual(
+                    "Activation for your Author's Heaven account",
```

Comment:
@codjoero Typing error on Author's Haven, It's Haven not Heaven

Reply:
resolved, thanks